src/message/MessageView.js

```diff
@@ -34,7 +34,7 @@ const styles = StyleSheet.create({
     fontSize: 15,
   },
   messageTime: {
-    color: '#999',
+    color: BORDER_COLOR,
```

Comment:
This is much lighter than it used to be. And I don't think its style should be related `BORDER_COLOR` in any case